package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@remoteoss/json-schema-form",
-  "version": "0.11.12-beta.0",
+  "version": "0.11.13-dev.20250502172456",
```

Comment:
@dragidavid to answer your comment above: [CI failing](https://github.com/remoteoss/json-schema-form/actions/runs/14800379574/job/41557512457?pr=180) as expected because of this change to the package.json. It's a reminder to revert it before merging the PR (and after the internal E2Es passes). Doing that now with the script `version_as_main`

Reply:
Done! [6e55485](https://github.com/remoteoss/json-schema-form/pull/180/commits/6e554852d73361ea665ecfddfb8edb9f344e4f67) can you give a final review?